src/pages/policies/[policyId]/items/new.svelte

```diff
@@ -18,35 +26,40 @@ let open = false
 onMount(() => {
   loadDependents(policyId)
   loadMembersOfPolicy(policyId)
+  loadItems(policyId)
 })
 
 $: metatags.title = formatPageTitle('Items > New')
 
-$: policyId && loadItems(policyId)
-
+$: item.id && $redirect(itemsNewQs(policyId, { itemId: item.id }))
 $: $selectedPolicy.type === PolicyType.Household && !$selectedPolicy.household_id && (open = true)
 
+$: $params.itemId && (item = $selectedPolicyItems.find((i) => i.id === $params.itemId) || {})
 $: breadcrumbLinks = [
   { name: 'Items', url: itemsRoute(policyId) },
-  { name: 'New', url: itemsNew(policyId) },
+  { name: 'New', url: item.id ? itemsNewQs(policyId, { itemId: item.id }) : itemsNew(policyId) },
```

Comment:
If you wanted, you could simplify this down as follows (since Item ID is the only piece of data it can include), though you'd also need to update the route helper accordingly:
```suggestion
  { name: 'New', url: item.id ? itemsNewQs(policyId, item.id) : itemsNew(policyId) },
```
However, that's effectively undoing my earlier recommendation (and changing some other things), so feel free to use your judgment. It just seemed like we ended up an an unnecessarily-complicated syntax for calling this route helper function.